lib/tree_sitter/node.rb

```diff
@@ -163,5 +163,94 @@ def fetch(*keys)
       end
       fields.values_at(*keys)
     end
+
+    # Regex for line annotatin extraction from sexpr with source.
```

Comment:
annotation.

Reply:
Disappointed with `typos`. Didn't catch this one.